src/styles/stars.scss

```diff
@@ -0,0 +1,21 @@
+.stars {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 16px;
+  padding: 0;
+  margin: 0;
+
+  &__star {
+    background-position: 50%;
+    height: 16px;
+    width: 16px;
+    background-image: url(../images/star.svg);
+    margin-right: 4px;
+    background-repeat: no-repeat;
+
+    &:nth-child(-n + 4) {
+      background-image: url(../images/star-active.svg);
+    }
```

Comment:
The use of ':nth-child(-n + 4)' selector is correct for applying the active star background to the first four stars. Ensure the path to 'star-active.svg' is correct and the image is available in the specified directory.